.github/workflows/mariner2.yml

```diff
@@ -23,24 +24,30 @@ jobs:
         with:
           submodules: 'recursive'
           set-safe-directory: true
-      - name: install dependencies
+      - name: Install dependencies
         run: |
           bash .install/mariner2.sh
-      - name: get redis latest
-        run: |
-          bash .install/get_redis.sh
-      - name: build and test
+      - name: Get Redis
+        uses: actions/checkout@v4
+        with:
+          repository: redis/redis
+          ref: '7.2.1'
+          path: redis
+      - name: Build Redis
+        working-directory: redis
+        run: make install && cd ..
```

Comment:
What's the ` && cd ..` for? By specifying `working-directory` for the step you don't have to worry about it in the next steps